### ISSUE TITLE
New pyright version availabile

### ISSUE THREAD

**Issue by schorlton** [oxsecurity/megalinter#2116]
Running `oxsecurity/megalinter-python:v6.15.0` in docker. Getting (without any additional error in the log file):

```bash
Results of pyright linter (version 1.1.270)
See documentation on https://megalinter.io/6.15.0/descriptors/python_pyright/
-----------------------------------------------

❌ [ERROR] for workspace /tmp/lint
Linter raw log:
WARNING: there is a new pyright version available (v1.1.270 -> v1.1.281).
Please install the new version or set PYRIGHT_PYTHON_FORCE_VERSION to `latest`

```

**Comment by nvuillam** [comment]
Pyright has been downgraded on purpose, because there were crashes with higher versions

https://github.com/oxsecurity/megalinter/blob/de03e7c5080d0aa8be32041bc24b292b103124fc/megalinter/descriptors/python.megalinter-descriptor.yml#L319

That's probably something that will be solved when we'll upgrade MegaLinter python version, and to do that we're waiting for a new release of multiprocessing-logging because of issue https://github.com/jruere/multiprocessing-logging/issues/56 ( that should happen this week )

**Comment by bdovaz** [comment]
Resolved in #2333
